dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1507,10 +1500,12 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 				force = true;
 			}
 
-			if (args.size() > index && !GeneralUtils::TryParse(args[index], cloneId)) {
+			const auto cloneIdOptional = GeneralUtils::TryParse<LWOCLONEID>(args[index]);
+			if (args.size() > index && !cloneIdOptional) {
```

Comment:
index check was changed to be after the access here

Reply:
Not sure how to avoid that while also getting the return value